v3/src/data-interactive/handlers/attribute-handler.ts

```diff
@@ -33,7 +34,9 @@ export const diAttributeHandler: DIHandler = {
       return { success: true }
     }).filter(error => !error.success)
     if (attributeErrors.length > 0) return attributeErrors[0]
-
+    if (!metadata && !Array.isArray(values) && 'colormap' in values && values.colormap) {
+      return sharedCaseMetadaNotFoundResult
```

Comment:
I'm afraid this error message is going to be confusing and unhelpful. Hopefully it will never occur because the `metadata` is actually always found. But if it is missing, there's no indication that it's the color map portion of the request that requires metadata. This message should say something like "SharedCaseMetadata not found but required for colormaps". Even that is confusing--will plugin developers have any idea what SharedCaseMetadata is?

Reply:
I suggested another option elsewhere.